website/docs/reference/resource-configs/bigquery-configs.md

```diff
@@ -764,26 +768,27 @@ Views with this configuration will be able to select from objects in `project_1.
 
 The `grant_access_to` config is not thread-safe when multiple views need to be authorized for the same dataset. The initial `dbt run` operation after a new `grant_access_to` config is added should therefore be executed in a single thread. Subsequent runs using the same configuration will not attempt to re-apply existing access grants, and can make use of multiple threads.
 
-<VersionBlock firstVersion="1.6">
-
+<VersionBlock firstVersion="1.7">
 
 ## Materialized view
```

Comment:
We don't support this yet for `dbt-bigquery`. But Florian wants this as soon as possible, via a patch release if necessary.